deploy/LOCAL_INSTALL.md

```diff
@@ -85,8 +90,23 @@ This is expected to take a while
     
    docker-compose exec pipeline-runner copy_reference_data_to_gs.sh $BUILD_VERSION $GS_BUCKET
    
-   ``` 
+   ```
+   Periodically, you may want to update the reference data in order to get the latest versions of these annotations. 
+To do this, run the following commands to update the data. All subsequently loaded data will then have the updated 
+annotations, but you will need to re-load previously loaded projects to get the updated annotations.
+   ```bash
+   GS_BUCKET=gs://your-bucket       # your google bucket
+   BUILD_VERSION=38                 # can be 37 or 38
    
+   # Update clinvar 
+   gsutil rm -r "${GS_BUCKET}/reference_data/GRCh${BUILD_VERSION}/clinvar.GRCh${BUILD_VERSION}.ht"
+   gsutil rsync -r "gs://seqr-reference-data/GRCh${BUILD_VERSION}/clinvar/clinvar.GRCh${BUILD_VERSION}.ht" "${GS_BUCKET}/reference_data/GRCh${BUILD_VERSION}/clinvar.GRCh${BUILD_VERSION}.ht"
+  
+   # Update all other reference data
+   gsutil rm -r "${GS_BUCKET}/reference_data/GRCh${BUILD_VERSION}/combined_reference_data_grch${BUILD_VERSION}.ht"
+   gsutil rsync -r "gs://seqr-reference-data/GRCh${BUILD_VERSION}/all_reference_data/combined_reference_data_grch${BUILD_VERSION}.ht" "${GS_BUCKET}/reference_data/GRCh${BUILD_VERSION}/combined_reference_data_grch${BUILD_VERSION}.ht"
```

Comment:
For local installation, the `gsutil rsync` command should sync to a bucket other than `gs://seqr-reference-data`.

Reply:
But thats where they get the data from. Its rsync'ing the content of that public bucket to the local machine which is exactly whats done in the helper scripts we tell them to run above

Reply:
I see. The `gs://seqr-reference-data` is accessible to the public.